src/back/TicketManager/src/Core/TicketManager.Core.Services/Endpoints/Events/ListEventsEndpoint.cs

```diff
@@ -0,0 +1,35 @@
+using FastEndpoints;
+using TicketManager.Core.Contracts.Common;
+using TicketManager.Core.Contracts.Events;
+using TicketManager.Core.Domain.Accounts;
+using TicketManager.Core.Services.DataAccess;
+using TicketManager.Core.Services.DataAccess.DtoMappers;
+using TicketManager.Core.Services.Extensions;
+
+namespace TicketManager.Core.Services.Endpoints.Events;
+
+public class ListEventsEndpoint : Endpoint<ListEventsRequest, PaginatedResponse<EventDto>>
+{
+    private readonly CoreDbContext dbContext;
+
+    public ListEventsEndpoint(CoreDbContext dbContext)
+    {
+        this.dbContext = dbContext;
+    }
+
+    public override void Configure()
+    {
+        Get("/event/list");
+        Roles(AccountRoles.User);
```

Comment:
I know we talked that only `User` should have access to this endpoint, but documentation says that every role should have access. So until we change specification allow all roles